.github/workflows/test.yml

```diff
@@ -458,12 +454,83 @@ jobs:
     needs: build_test_kit_docker
     runs-on: ubuntu-22.04
     steps:
-      - uses: manticoresoftware/clt@0.1.13
+      - uses: manticoresoftware/clt@0.1.5
         with:
           artifact: manticore_test_kit.img
           image: test-kit:img
           test_prefix: test/clt-tests/expected-errors/
 
+  clt_sharding_cluster_test:
+    name: CLT test cluster logic of the manticoresearch
+    needs: build_test_kit_docker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: manticoresoftware/clt@0.1.5
+        with:
+          artifact: manticore_test_kit.img
+          image: test-kit:img
+          test_prefix: test/clt-tests/sharding/cluster/
+          run_args: -e TELEMETRY=0
```

Comment:
Remove this line.

Reply:
And all other occurrences of `-e TELEMETRY=0`. Anyway the env. vars. don't make it to the CLT working container.